src/assets/i18n/it.po

```diff
@@ -4164,10 +4179,10 @@ msgid "Decrypting Disks"
 msgstr "Decodifica Dischi"
 
 msgid "Dedup"
-msgstr ""
+msgstr "Deduplica"
 
 msgid "Dedup VDev"
-msgstr ""
+msgstr "Vdev Deduplica"
```

Comment:
Sarebbe "Deduplicata"?

Reply:
Umm! Non so, ma è sbagliata : )
Forse intendono "Deduplica vdev" o "Vdev di deduplica"? Sul man non trovo corrispondenza, quindi credo sia un messaggio di help.

direi di tenerlo in sospeso e prendere una decisione dopo il test sulla VM.

Si può lasciare in _open_ su issue in attesa di verifica.

Reply:
La deduplicazione è intesa quando si decomprime un volume compresso, è una particolarità di zfs anche la compressione è particolare, il termine deduplica deve rimanere tale, in italiano va bene. La traduzione corretta è “ Deduplica Vdev”